experimental/README.md

```diff
@@ -9,65 +9,22 @@ issues associated with it. If necessary, links are provided to additional
 documentation on an issue.  As an active Docker user and community member,
 please feel free to provide any feedback on these features you wish.
 
-## Install Docker experimental
+## Use Docker experimental
 
-Unlike the regular Docker binary, the experimental channels is built and
-updated nightly on https://experimental.docker.com. From one day to the
-next, new features may appear, while existing experimental features may be
-refined or entirely removed.
+Experimental features are now included in the standard Docker binaries as of
+version 1.13.0.
+For enabling experimental features, you need to start the Docker daemon with
+`--experimental` flag.
+You can also enable the daemon flag via `/etc/docker/daemon.json`. e.g.
 
-1. Verify that you have `curl` installed.
+        {
```

Comment:
looks like this is still using 8 spaces for indentation (which is no longer needed because the numbered list is gone).
while updating, can you use "fences" for this block, and a `JSON` code-hint? i.e.

    ```json
    {
        "experimental":true
    }
    ```